plugins/Identify.jsx

```diff
@@ -304,8 +307,8 @@ class Identify extends React.Component {
             return {identifyResults: identifyResults};
         });
     };
-    onShow = (mode) => {
-        this.setState({mode: mode || 'Point'});
+    onShow = (mode, data) => {
+        this.setState({mode: mode || 'Point', clickPoint: data.pos, exitTaskOnResultsClose: data.exitTaskOnResultsClose});
```

Comment:
I think it could work in an even cleaner way by:
```
 if (this.props.currentIdentifyTool === "Identify" || this.props.currentTask === "Identify") {
             if (this.state.mode === "Point") {
+               const clickPoint = this.queryPoint(prevProps);
-               this.identifyPoint(prevProps);
+               this.identifyPoint(clickPoint);
             } else if (this.state.mode === "Region") {
```

and the modify `identifyPoint` below
```
-    identifyPoint = (prevProps) => {
+    identifyPoint = (clickPoint) => {
```
and invoke `identifyPoint(data.pos)` directly here instead of storing the `clickPoint` in the state.